__tests__/components/BB_test.js

```diff
@@ -200,7 +204,9 @@ describe("BB", () => {
         { patronType: "service-person" },
         { need1: "need1", need2: "need2" },
         "sorting",
-        "en"
+        "en",
+        "0",
+        "0"
       );
     expect(url).toEqual(
       "print?lng=en&patronType=service-person&needs=need1,need2&sortBy=sorting&benefits=id1,id2"
```

Comment:
How is this test passing? Should it not be `&closestAOID=0&selectedAOID=0`

Reply:
two wrongs make a right? The print function actually expects the area office objects, not their ids as I was sending.

Reply:
(btw, thanks - good catch!)

Reply:
The sad thing is that Javascript just went at head with "0" as undefined because it should have appended just the empty GET names.

Reply:
actually the code happily computes `"0".id`, got undefined, went its merry way.